packages/gatsby-plugin-sharp/README.md

```diff
@@ -242,12 +250,21 @@ You can opt-in to use [MozJPEG][16] for jpeg-encoding. MozJPEG provides even
 better image compression than the default encoder used in `gatsby-plugin-sharp`.
 However, when using MozJPEG the build time of your Gatsby project will increase
 significantly.
-To enable MozJPEG set the [environment variable](/docs/environment-variables/#environment-variables):
+To enable MozJPEG, you can set the `useMozJpeg` plugin option to `true` in
+`gatsby-config.js` or set
+the [environment variable](/docs/environment-variables/#environment-variables):
 
 ```shell
 GATSBY_JPEG_ENCODER=MOZJPEG
 ```
 
+### EXIF and ICC metadata
+
+By default, `gatsby-plugin-sharp` strips all EXIF, ICC and other metadata
+present in your source file. If you wish to preserve this metadata, you can
```

Comment:
Is it helpful to document _why_ you'd want to keep this here?

Reply:
I guess it doesn't hurt to add that information now that I think about it.